src/flatbuffers/Expr.fbs

```diff
@@ -118,6 +119,15 @@ table Contains {
     right:Expr;
 }
 
+// Array expressions
+table In{
+    children:[Expr]; 
+}
```

Comment:
This definition shouldn't appear in this PR, since it's already in the IN PR.

Reply:
Removed